docs/DeveloperGuide.md

```diff
@@ -1,38 +1,152 @@
 # Developer Guide
 
 ## Acknowledgements
-
 {list here sources of all reused/adapted ideas, code, documentation, and third-party libraries -- include links to the original source as well}
+* Java Coding Standard (Basic): [Official Documentation](https://se-education.org/guides/conventions/java/basic.html)
+* Code Quality (Guide): (https://nus-cs2113-ay2425s1.github.io/website/se-book-adapted/chapters/codeQuality.html)
 
 ## Design & implementation
-
 {Describe the design and implementation of the product. Use UML diagrams and short code snippets where applicable.}
 
+### "Add" Command Feature
```

Comment:
Inconsistent formatting styles: "Add" (line 11),  '-h' (line 15), `command` (line 20). It is recommended that you use a single style for code snippets, most preferably `...`.